intelmq/bots/collectors/fireeye/collector_fireeye.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+"""
+Fireeye collector bot
+
+Parameters:
+http_username, http_password: string
+http_timeout_max_tries: an integer depicting how often a connection attempt is retried
+dns_name : dns name of the local machine
+request_duration: how old date should be fetched eg 24_hours or 48_hours
+"""
+import base64
+import json
+
+from intelmq.lib.bot import CollectorBot
+from intelmq.lib.utils import unzip, create_request_session_from_bot
+
+
+try:
+    import requests
+    import xmltodict
+except ImportError:
+    requests = None
+
+
+class FireeyeCollectorBot(CollectorBot):
+
+    def xml_processor(self, uuid, token, new_report, dns_name, product):
+        with requests.Session() as s:
+            url = 'https://' + dns_name + '/wsapis/v2.0.0/openioc?alert_uuid=' + uuid
+            headers = {'X-FeApi-Token': token}
+            r = requests.get(url, headers=headers, verify=False)
+            binary = r.content
+            self.logger.info('collecting information for uuid :' + uuid)
```

Comment:
For all logging calls, please use this form:
```python
self.logger.info('Collecting information for UUID: %r', uuid)
```
That results in better error handling by the logging facilities in case of issues with the parameter